pandas/tests/indexing/multiindex/test_loc.py

```diff
@@ -623,3 +623,10 @@ def test_getitem_non_found_tuple():
     )
     with pytest.raises(KeyError, match=r"\(2\.0, 2\.0, 3\.0\)"):
         df.loc[(2.0, 2.0, 3.0)]
+
+
+def test_get_loc_datetime_index():
+    # GH: 18185
+    index = pd.date_range("2001-01-01", periods=100)
+    mi = MultiIndex.from_arrays([index])
+    assert mi.get_loc("2001-01") == slice(0, 31, None)
```

Comment:
i dont know if this is wrong, but it is definitely weird.  i would expect `mi.get_loc("2001-01")` to match `index.get_loc("2001-01")` here.  why doesnt it?

Reply:
It does match, just wasn't tested. I added a line to test that too.

Reply:
Somehow my respone from yesterday got lost. Forgot to press sent probably :)

Bot results match, did not test ``index.get_loc("2001-01")`` previously but added it

Reply:
huh, dont know why i thought they didnt match.  can you add a comment about why this assertion is here

Reply:
Done